ui/app/routes/vault/cluster/access/method.js

```diff
@@ -12,7 +15,10 @@ export default Route.extend({
         set(error, 'httpStatus', 404);
         throw error;
       }
-      return model;
+      return this.pathHelp.getPaths(model.apiPath, path).then(paths => {
+        model.set('paths', paths);
```

Comment:
(for later) we should document what attributes OpenAPI models have on them - `paths`, `newFields`, etc.